src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
Cuda 7.0 defines also a `pow(float,float)` and `pow(float,int)` but it might not be documented online.

math_functions.hpp:103-105
```C
__host__ __device__ __cudart_builtin__ float     pow(float a, float b)   { return powf(a, b); }
__host__ __device__ float powif(float, int);
__host__ __device__ __cudart_builtin__ float     pow(float a, int b)     { return powif(a, b); }
```